### PR TITLE
Thermomachine pipe color change

### PR DESCRIPTION
**Pull request by Ghilker**
## About The Pull Request
This PR allow the players to change the thermomachines pipe color with the multitool right click when it's unanchored.
plus some fixes for hitting the machine with tools 

## Why It's Good For The Game
players can now connect the machine to colors other than grey

## Changelog
:cl:
qol: players can change the thermomachines pipe color with the multitool right click when it's unanchored.
fix: fixed hitting the machine with tools in some cases
/:cl:

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
Why would you need to do this in the first place? pipes and components have color for ease of use, why do thermo-machines need the system?

**Comment by Ghilker** [comment]
> Why would you need to do this in the first place? pipes and components have color for ease of use, why do thermo-machines need the system?

because they currently can connect to every pipe since they are grey, this is in case of cramped spaces with lots of pipes around you, this allows you to prevent accidents and wrongful connections

File: code/modules/atmospherics/machinery/components/binary_devices/thermomachine.dm [hunk 1]
```diff
@@ -220,7 +224,13 @@
 		investigate_log("had its internal [holding] swapped with [tank] by [key_name(user)].", INVESTIGATE_ATMOS)
 		replace_tank(user, tank)
 		update_appearance()
+		return
 
+	if(!anchored && item.tool_behaviour == TOOL_MULTITOOL)
```
Comment (LemonInTheDark) [inline code/modules/atmospherics/machinery/components/binary_devices/thermomachine.dm:229]: why !anchored?

Comment (Ghilker) [reply code/modules/atmospherics/machinery/components/binary_devices/thermomachine.dm:229]: yeah i'll make it with only the panel open

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**